arangod/Cluster/CreateCollection.cpp

```diff
@@ -165,7 +165,7 @@ bool CreateCollection::first() {
         std::vector<std::string> noFollowers;
         col->followers()->takeOverLeadership(noFollowers, nullptr);
       } else {
-        col->followers()->setTheLeader(leader);
+        col->followers()->setTheLeader(LEADER_NOT_YET_KNOWN);
```

Comment:
It was a very bad idea to create a shard which is not yet in sync yet with the leader already configured.